### PR TITLE
scala app service sample

### PR DESCRIPTION
**Pull request by denvermb**

### CODE REVIEW

**Comment by JasonFreeberg** [comment]
## [Preview link](https://antares-blog-staging-pr-313.azurewebsites.net) 
- Your changes have been deployed to the preview site. The preview site will update as you add more commits to this branch. 
- The preview site shows any future-dated articles. Don't worry, if you are publishing a future-dated article, it will not show on the production site until the file's specified date.
- The preview link is shareable, but will be deleted when the pull request is merged or closed.
> *This is an automated message.*

File: _posts/2022-05-31-scala-on-app-service.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: "Scala on App Service"
+toc: true
+toc_sticky: true
+author_name: Denver Brittain
+excerpt: "Run a Scala App using Java SE on App Service"
+category: java
+---
+
```
Comment (JasonFreeberg) [inline _posts/2022-05-31-scala-on-app-service.md:9]: Can you add an introduction here? Introduce concepts (Scala, Play framework) and then use the generic, _follow the tutorial below to deploy a Play framework Scala app onto Azure App Service._ 

Really helps with SEO ;)

File: _posts/2022-05-31-scala-on-app-service.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+To check your sbt version, enter the following in a command window:
+
+```bash
+sbt sbtVersion
+```
+
+If you do not have the required versions, follow these links to obtain them:
+
+* [Java SE](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
```
Comment (JasonFreeberg) [inline _posts/2022-05-31-scala-on-app-service.md:35]: Replace Oracle JDK link with MSFT JDK link:

```suggestion
* [Java SE](https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11)
```

File: _posts/2022-05-31-scala-on-app-service.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+* [sbt](http://www.scala-sbt.org/download.html)
+* [maven](https://maven.apache.org/install.html) (Optional)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) (Optional)
```
Comment (JasonFreeberg) [inline _posts/2022-05-31-scala-on-app-service.md:38]: you can move these hyperlinks into the bullets on lines 16-19, cuts down on length

File: _posts/2022-05-31-scala-on-app-service.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    ```
+
+    This command produces an executable .jar file in the scala-on-app-service/target/scala-2.14 directory.
+
+2. To test the app locally, run the previously created .jar file:  
+
+    ```bash
+    java -jar target/scala-2.13/<project-name>-assembly-<version>.jar
+    ```
+
```
Comment (JasonFreeberg) [inline _posts/2022-05-31-scala-on-app-service.md:74]: Add a "Step 3" to check the app in the browser again

Add a small paragraph that gives the high-level technicalities of the fat JAR. (You explained it in our 1:1 before, like how the "assembly" command includes the whole Scala runtime or something to that effect)

**Review by JasonFreeberg — CHANGES_REQUESTED** [review]

File: _posts/2022-05-31-scala-on-app-service.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+## Prerequisites
+
+This sample is based off of the [Play Framework Hello World Tutorial](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-hello-world-tutorial). 
+
+To follow the steps in this tutorial, you will need the correct version of Java, sbt, and (optionally) Maven or the Azure CLI.
+
+* Java 11
+* sbt 1.3.4 or higher. Note: if you downloaded this project as a zip file from <https://developer.lightbend.com>, the file includes an sbt distribution for your convenience.
+* Maven (Optional)
```
Comment (dfuenzalida) [inline _posts/2022-05-31-scala-on-app-service.md:18]: If you are using Maven commands, I think you need to remove the Optional label for Maven. Same on line 14.

**Review by JasonFreeberg — APPROVED** [review]

**Pull request merged**